app/js/classAdd.controller.js

```diff
@@ -1,5 +1,5 @@
 export default angular
-    .module('conceptDictionaryApp')
+    .module('stuff')
```

Comment:
Separate modules should be created for each controller (a piece of functionality). Let's name it conceptDictionaryApp.classAdd. Apply the same pattern to other modules. The "conceptDictionaryApp." part indicates that it is a submodule of conceptDictionaryApp.